### ISSUE TITLE
Class names get lowercased in Safari 9, iOS 9

### ISSUE THREAD

**Issue by max-mathieu** [DylanPiercey/set-dom#21]
This bug is pretty crazy, and it took us (dev team @CollageCom) a while to figure out what was the issue.

In Safari 9 and iOS 9 (at least) `DOMParser.parseFromString()`-generated elements are getting their classes lowercased when injected in the DOM. This breaks CSS rules and any JS code using classes to bind event listeners and other actions.

In action, in Safari 9's console:
```
> p = new DOMParser();
> a = p.parseFromString('<div class="A"></div>').body.firstChild;
> a.classList[0]
"A"
> document.body.appendChild(a);
> a.classList[0]
"a"
```
Safari 10 and iOS 10 are fine

PR is coming shortly with a fix

**Comment by DylanPiercey** [comment]
Gotta love browsers!

Excellent work tracking this down, going to review and test things out right now!

**Comment by DylanPiercey** [comment]
Published in 7.4.5 🤠, thanks for your help!

**Comment by max-mathieu** [comment]
You're welcome, thanks for maintaining this lib and be so reactive!
